pkg/sdk/parsers.go

```diff
@@ -37,3 +37,18 @@ func ParseCommaSeparatedStringArray(value string, trimQuotes bool) []string {
 	}
 	return trimmedListItems
 }
+
+// ParseCommaSeparatedSchemaObjectIdentifierArray can be used to parse Snowflake output containing a list of schema-level object identifiers
+// in the format of ["db".SCHEMA."name", "db"."schema2"."name2", ...],
+func ParseCommaSeparatedSchemaObjectIdentifierArray(value string) ([]SchemaObjectIdentifier, error) {
```

Comment:
nit: could be written in a more generic way for us to extend the functionality for other id types

Reply:
It could, but it's not needed for now.